cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/package-info.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+/**
+ * This package contains the DataCleansing sample Application filters records that do not match a given schema.
```

Comment:
the DataCleansing Application that filters records that do not match a given schema.